lib/daffy_lib/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module DaffyLib
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```

Comment:
Can we leave version as 0.1.0. We haven't had any releases yet, so no need to rev version number.

Reply:
I did push it to RubyGems actually: https://rubygems.org/gems/daffy_lib